### PR TITLE
Admin Tools - Audit + Permissioning Documentation

### PR DESCRIPTION
**Pull request by GanadiniAkshay**

### CODE REVIEW

File: website/pages/en/index.js [hunk 1]
```diff
@@ -70,8 +70,14 @@ module.exports = () => (
           description:
             'Learn how to uncover hidden insights in your chat data & communication',
           ctaText: 'Get Started',
-          ctaLink: '/bot-analytics',
+          ctaLink: '/bot-analytics'
         },
+        {
+          title: 'Admin Tools',
+          description: 'Manage the users, data and other settings for the paltform',
```
Comment (medhaanand) [inline website/pages/en/index.js:77]:
```suggestion
          description: 'Manage the users, data and other settings for the platform',
```

File: docs/admin-tools/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Introduction
+---
+
+The Admin Tools of the Haptik Platform are a set of tools that help you manage the users, the data and other configurations of all others tools on the platform like Bot Builder and Live Chat.
```
Comment (medhaanand) [inline docs/admin-tools/index.md:5]:
```suggestion
The Admin Tools of the Haptik Platform are a set of tools that help you manage the users, the data and other configurations of all others tools on the platform like Bot Builder, Agent Chat and Chat Analytics.
```

File: docs/admin-tools/audit-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+**Note**: The Audit Dashboard is only available to Partner Admins.
+
+![Audit Dashbaord](assets/audit-dashboard.png)
+
+## Using the Audit Dashboard
+
+* Partner Admins can select the partner, the tool and the date period for which they want to get the data.
+![Audit Request](assets/audit-request.gif)
+
+* The Partner Admin will then recieve an email with a encoded zip file of the audit data along with the password to access the zip file.
```
Comment (medhaanand) [inline docs/admin-tools/audit-dashboard.md:16]:
```suggestion
* Partner Admin will then recieve an email with a encoded zip file of the audit data along with the password to access the zip file.
```

File: docs/admin-tools/audit-dashboard.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+The Audit Dashboard tool allows users to download audit logs for each of the platform tools for a specified period of time. These audit logs give you information about what changes were made and by whom.
+
+**Note**: The Audit Dashboard is only available to Partner Admins.
+
+![Audit Dashbaord](assets/audit-dashboard.png)
+
+## Using the Audit Dashboard
+
+* Partner Admins can select the partner, the tool and the date period for which they want to get the data.
```
Comment (medhaanand) [inline docs/admin-tools/audit-dashboard.md:13]:
```suggestion
* Partner Admin can select the partner, the tool and the date period for which they want to get the data.
```

File: docs/admin-tools/permission-management.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+The Permission management tool allows Partner Admins to manage the partners they have access to and provides the following functionality:
+
+* Create New Partner
+* Add Existing Users to a partner
+* Create New User inside a partner
+* Change Roles for the user
+* Make Bulk Addition/Updates to the users
+
+All of the above functionality is only available to a Partner Admin.
```
Comment (medhaanand) [inline docs/admin-tools/permission-management.md:13]:
```suggestion
> All of the above functionality is only available to a Partner Admin.
```

File: docs/admin-tools/permission-management.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+## What is a Partner Admin?
+A Partner Admin is a special role that can be assigned to someone who needs to manage the access and roles for all users under a partner.
+
+## Available Roles in the System
+The roles are divided or categorised basis the tools they give access to. The available roles in the system are as follows:
```
Comment (medhaanand) [inline docs/admin-tools/permission-management.md:19]:
```suggestion
The roles are divided or categorised on the basis of tools. The available roles in the system are as follows:
```

File: docs/admin-tools/audit-dashboard.md (hunk 3 above)
Comment (GanadiniAkshay) [reply docs/admin-tools/audit-dashboard.md:16]: Grammatically it needs to be "The Partner Admin" only

**Pull request merged**